schema/validator.go

```diff
@@ -46,17 +47,18 @@ func (e ValidationError) Error() string {
 }
 
 // Validate validates the given reader against the schema of the wrapped media type.
-func (v Validator) Validate(src io.Reader) error {
+// strict: accept the customized media type when value is false, or reject when true.
```

Comment:
fixed in https://github.com/opencontainers/image-spec/pull/452/commits/b68d69226243c79b275a3c5caeac04e3dad84231#diff-4a672c41642ea30ebf88bb87685cf72aR50